lib/ssh_data/private_key/rsa.rb

```diff
@@ -21,9 +21,9 @@ def self.generate(size, unsafe_allow_small_key: false)
 
       # Import an openssl private key.
       #
-      # key - An OpenSSL::PKey::DSA instance.
+      # key - An OpenSSL::PKey::RSA instance.
       #
-      # Returns a DSA instance.
+      # Returns a RSA instance.
```

Comment:
I thought about using "an" instead of "a" here but wanted to be consistent with the ed25519 phrasing (https://github.com/github/ssh_data/blob/master/lib/ssh_data/private_key/ed25519.rb#L18)